float32 exposureTime
---
float32 currentExposureTime
